SQL_Assignment_3.sql

```diff
@@ -0,0 +1,245 @@
+--1. Extract the middle 3 characters from the string 'ABCDEFG'.
+
+SELECT SUBSTRING('ABCDEFG',3,3)
+
+--2 2. From a table 'Employees' with a column 'FullName', write a query
+-- to extract the first name (assuming it's always the first word before a space).
+
+SELECT LEFT(FULLNAME, CHARINDEX(' ',FULLNAME,0)) AS FIRST_NAME
```

Comment:
Why 0 at the end ? Also, the output will contain space at the end, right ?

Reply:
To indicate the start position
yes, output will contain space at the end, i have to indicate left and subtract 1 from the index of space.
corrections made.